README.md

```diff
@@ -75,6 +75,15 @@ The default timeout for your Headless JS task is 5 minutes (300000ms). If you wa
 <string name="rnsb_default_timeout" translatable="false">600000</string>
 ```
 
+### User visible
+
+By default the user will not be able to manually request a sync from the settings. If you want the user to have the option to request a sync from the settings, you have to override `strings.xml` again:
```

Comment:
I think we need to fix the description. For what I see, this is the difference: https://stackoverflow.com/a/25960728/2277631. So basically, when we set it to `true`, we allow the user to disable/enable the sync functionality using the Account settings. Can we change the description to:

> By default the user will not be able to manually enable/disable syncs through Account settings. If you want the user to have this option, you need to override `strings.xml`: